web/datapipeline/clusters_projector.go

```diff
@@ -36,6 +36,10 @@ func clustersProjector_ClusterDiscoveryHandler(event *DataCollectedEvent, db *go
 		return err
 	}
 
+	if !cluster.DC {
+		return nil
```

Comment:
Could we add a debugging trace here saying `Cluster node is not a DC. Skipping projection` or similar

Reply:
done